soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3956,6 +3959,14 @@ void DrawRandoEditor(bool& open) {
                                             "expected to be collected after getting Sun's Song.");
                         PaddedSeparator();
 
+                        // Shuffle Cows
+                        ImGui::Text(Settings::ShuffleCows.GetName().c_str());
+                        InsertHelpHoverText(
+                            "Cows give a randomized item from the pool upon performing Epona's\n"
```

Comment:
we have logic to automatically add newlines to tooltips now, so the `\n` should be removed